package.json

```diff
@@ -64,13 +64,13 @@
     "express": "4.16.3",
     "ink-docstrap": "1.3.2",
     "jsdoc": "3.5.5",
-    "jsonwebtoken": "8.2.1",
-    "marked": "0.3.19",
+    "jsonwebtoken": "8.3.0",
+    "marked": "0.4.0",
     "method-override": "2.3.10",
     "mime": "1.6.0",
     "minimatch": "3.0.4",
     "mongo-uri": "0.1.2",
-    "mongodb": "2.2.33",
+    "mongodb": "2.2.35",
```

Comment:
Did you check out? The latest https://www.npmjs.com/package/mongodb ?
Could consider an update..